doc/lua_api.txt

```diff
@@ -1010,22 +1010,24 @@ The function of `param2` is determined by `paramtype2` in node definition.
     * Values range 0 - 179. The value stored in `param2` is multiplied by two to
       get the actual rotation in degrees of the node.
 * `paramtype2 = "meshoptions"`
-    * Only valid for "plantlike" drawtype. The value of `param2` becomes a
-      bitfield which can be used to change how the client draws plantlike nodes.
-    * Bits 0, 1 and 2 form a mesh selector.
-      Currently the following meshes are choosable:
+    * Only valid for "plantlike" drawtype. `param2` encodes the shape and
+      optional shape modifiers of the "plant". `param2` is a bitfield.
```

Comment:
Here and below:

Sorry to be pedantic but, bits 3 to 7 are mostly not 'shape modifiers', they are size or position modifiers.
Maybe just 'optional modifiers'?

Reply:
Done.